frontend/src/components/GifWidget/GifWidget.styled.ts

```diff
@@ -0,0 +1,89 @@
+import styled from "styled-components";
+
+const GifWidgetStyled = styled.div`
+    position: absolute;
+    right: 3rem;
+    bottom: 7rem;
+
+    width: 30rem;
+    height: 35rem;
+    background-color: #000000bb;
+
+    box-shadow: 5px 10px 25px #000000bb;
+    border-radius: 4px;
+    overflow-y: auto;
+
+    &::-webkit-scrollbar {
+        width: 5px;
+    }
+
+    &::-webkit-scrollbar-thumb {
+        border-radius: 3px;
+        background: ${({ theme }) => theme.primary};
+    }
+`;
+
+const GifWidgetContent = styled.div``;
+
+const GifWidgetSearch = styled.input`
+    all: unset;
+
+    width: 16rem;
+    height: 3rem;
+    padding: 0 1rem;
+    margin: 1rem 1rem;
+
+    color: ${({ theme }) => theme.white};
+    background-color: ${({ theme }) => theme.gray_400};
+    border: 1px solid ${({ theme }) => theme.primary};;
+    border-radius: 0.2rem;
+    box-shadow: 5px 10px 25px #000000bb;
+`;
+
+const GifList = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    gap: 0.8rem;
+    height: auto;
+    justify-content: center;
+`;
+
+const GifListImage = styled.img`
+    width: 30%;
+    height: 8rem;
+    object-fit: cover;
+    border-radius: 4px;
+    cursor: pointer;
+`;
+
+const GifWidgetAtrtibutionSection = styled.div`
+    display: flex;
+    align-items: center;
+    gap: .5rem;
+    bottom: 7rem;
+
+    position: fixed;
+    width: 30rem;
+    padding: 0.7rem 1rem;
+
+    font-size: 13px;
```

Comment:
For font sizing  `rem` unit should be used.

Reply:
it's done